coordinator/src/main/java/org/apache/uniffle/coordinator/ApplicationManager.java

```diff
@@ -282,6 +328,58 @@ private String getStorageHost(String remoteStoragePath) {
     return storageHost;
   }
 
+  public void detectUserResource() {
```

Comment:
Should the application manager involve the quota logic?

Reply:
Because we can configure `rss.coordinator.quota.default.path` to controls whether to execute the logic of `detectUserResource`. If not configured, the logic will not be executed.

Reply:
Do we need a quota manager? Or will application manager  manage both quota and application?

Reply:
Maybe quota manager will be better.